nodemon.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
any particular reason you're removing this?

My thinking when adding it was most people would be using the `dev` script with nodemon,
so as they change code, it's automatically formatted.

Reply:
I see the reason but I was playing around some of the code and one of things that happens every once in a while is that standard encounters an issue that cannot be automatically fixed. One example which occurs often enough in local dev is unused variables.

When this happens the whole thing crashes a bunch of times before failing. I think it's much handier if we simply include linting as part of precommit hooks and as part of CI checks. Happy to keep it though if you prefer

Reply:
Could we make the pre-commit hook run `standard --fix`?
If it's just linting (and warnings/errors), there's nothing stopping badly formatted code getting into a PR.
In my experience, badly formatted code getting into a PR can distract from the actual goal of a PR (bikeshedding and such).
Lets fix it automatically so the review and discussion stays focused on the code, not formatting.

Reply:
I would advise against it. This is kind of difficult and dangerous to do because you would need to run `git add` again before the commit happens. How do you know what to add?

Tools that magically add stuff to `git commit` in the background are really dangerous IMO. In my experience, developers often selectively commit things and this kind of approach would lead to accidental files being committed.

I don't think this would distract reviewers too much because there are two protections in place. 1. when the developer runs `git commit` they won't be able to commit unless lint issues are fixed. 2. if they skip the precommit hook, the CI job will still fail on the linting also. I think this does a pretty good job ensuring reviewers don't need to waste time on this stuff.

Reply:
> I would advise against it. This is kind of difficult and dangerous to do because you would need to run git add again before the commit happens. How do you know what to add?

Understood.

So why not save the developer time and just fix formatting issues automatically before they even get to the `git add` phase?
The most likely thing they'll do after the git pre-commit fails is run the auto fix command anyways, so we can save them a step by doing it automatically.